src/analysis/rule_extraction.jl

```diff
@@ -123,13 +123,13 @@ A StableRules object (implemented by SIRUS).
    https://doi.org//10.1214/20-EJS1792
 """
 function cluster_rules(clusters::Vector{T}, X::DataFrame, max_rules::T;
-    n_trees::T=1000, seed=123) where {T<:Int64}
+    n_trees::T=1000, seed=123, lambda=1e-3) where {T<:Int64}
```

Comment:
Would it be more convenient to specify a generic `kwargs...` argument, given `lambda` is simply passed to SIRUS?

The upside is it will support any other args you might need that SIRUS has.

Reply:
Yes, that makes sense.

Reply:
Done